functions/uploader/config.js

```diff
@@ -1,21 +1,21 @@
 module.exports = {
   dfendpoints: {
-    DF_ENTITY_GET_URL: 'https://api.dialogflow.com/v1/entities/{{entityname}}?v=20150910',
-    DF_ENTITY_POST_URL: 'https://api.dialogflow.com/v1/entities/{{entityname}}/entries?v=20150910',
-    DF_ENTITY_DELETE_URL: 'https://api.dialogflow.com/v1/entities/{{entityname}}?v=20150910',
-  },
-  uploader: {
-    collection: {
-      ID: 'etree',
-      FIELD: 'creator',
-      LIMIT: 30000,
-      ENTITY: 'testing-collection',
-    },
-    genres: {
-      ID: 'georgeblood',
-      FIELD: 'subject',
-      LIMIT: 30000,
-      ENTITY: 'testing-genres',
+    DF_ENTITY_POST_URL: 'https://dialogflow.googleapis.com/v2/projects/internet-archive/agent/entityTypes/{{entityid}}/entities:batchUpdate?access_token={{accesstoken}}',
+    uploader: {
+      collection: {
+        ID: 'etree',
+        FIELD: 'creator',
+        LIMIT: 30000,
+        ENTITY: 'testing-collection',
```

Comment:
do we still need `ENTITY`?

Reply:
@hyzhak : It is not required yet but it is more likely possible that they might use start using entity name again in future. And this additional property is also increasing readability i.e. anyone can easily detect that with which entity this configuration is associated.

Reply:
Entity is the name of entity whereas Entity_Id is the unique id of that Entity. Previous version of DialogFlow api was working on both Entity/Entity_Id but v2 is supporting only Entity_Id.

Reply:
oh, I see. It looks like a regression on Dialog Flow side, do you know why they drop support of entity name? And how could we request entity id by its name?

Reply:
@hyzhak : Didn't find any find any clue for why they dropped but we can get the entity id using entity name via https://cloud.google.com/dialogflow-enterprise/docs/reference/rest/v2/projects.agent.entityTypes/list

Reply:
hm, yes I could work but could we ask DialogFlow team here?

Reply:
Ok sure, I will check this with DialogFlow team as well.

Reply:
@hyzhak : I received this response from DialogFlow Team for this query:

Sorry for the inconvenience. Currently Dialogflow V2 API method projects.agent.entityTypes.get to fetch the specified entity details will only accept entity id value in your request (but not entity name), please refer https://dialogflow.com/docs/reference/api-v2/rest/v2/projects.agent.entityTypes/get. And as you said, with V1 API, you can use GET /entities/{id} method to fetch the specified entity details by passing either entity id or entity name value in your request, please refer https://dialogflow.com/docs/reference/agent/entities#get_entitiesid.

Reply:
@hyzhak : Should I remove Entity_Name from here?